### ISSUE TITLE
zsh crashing upon cat $HOME/.local/bin/statusbar/battery

### ISSUE THREAD

**Issue by Jemi** [LukeSmithxyz/voidrice#450]
zsh is crashing on my artix laptop upon opening $HOME/.local/bin/statusbar/battery. It happens with cat, nvim and less. This also happens with bash.

**Comment by Jemi** [comment]
found this issue:
https://github.com/LukeSmithxyz/st/issues/125

The problem as it currently stands seems to be that there is currently no emojione available for arch/artix, and the symbola (and symbola-ib) packages do not work because of a license issue and a bad download link by the creator of that font.

Here's the workaround I found to keep st from crashing:

1) after yay fails to install ttf-symbola go to this link (found from aur discussion: https://aur.archlinux.org/packages/ttf-symbola/) and download symbola.zip:  https://fontlibrary.org/assets/downloads/symbola/cf81aeb303c13ce765877d31571dc5c7/symbola.zip

2) go to the cache directory created by yay: 
cd ~/.cache/yay/ttf-symbola

3) copy symbola package file into current directory: 
cp ~/Downloads/symbola.zip ./

4) change name of file to the one expected by yay: 
mv symbola.zip ttf-symbola-12.00.zip

5) install package, instructing yay to skip checks:
yay -S ttf-symbola --mflags "--skipchecksums --skippgpcheck"

After this, symbola should be installed and st will hopefully not crash anymore because of emojis.

BTW, this is really a LARBS issue. I'm opening an issue there about the need for LARBS to install 1 or more reliable emoji fonts.

**Comment by vladdoster** [comment]
This issue has been addressed in #421

**Comment by Jemi** [comment]
Opened this LARBS issue: https://github.com/LukeSmithxyz/LARBS/issues/157

421 doesn't really address this issue but merely works around it. The problem is that emojione is gone and symbola is not really reliable. There are licensing issues for both packages - meaning that although they could be useful for users who like the appearance of those fonts, including those fonts in larbs is asking for trouble. Unencumbered fonts providing the necessary coverage need to be identified and incorporated into LARBS.

**Comment by Jemi** [comment]
My bad. This issue was addressed there by LukeSmithxyz:

https://github.com/LukeSmithxyz/voidrice/issues/421#issuecomment-560504809

**Comment by vladdoster** [comment]
> My bad. This issue was addressed there by LukeSmithxyz:
> 
> [#421 (comment)](https://github.com/LukeSmithxyz/voidrice/issues/421#issuecomment-560504809)

All good. I hope you figure it out. I have personally moved to polybar and really love it. No emojis and still looks really clean.

![pic-selected-200113-2004-28](https://user-images.githubusercontent.com/10052309/72304922-1d8fed80-3640-11ea-9db5-9dbb112745c6.png)

**Comment by kronikpillow** [comment]
I guess people are having a hard time, understanding that they need to check previous issues for possible solutions before reposting something that was solved in the previous 20 issue posts by guys who didnt check it

**Comment by Jemi** [comment]
As far as I know, the issue is not solved in LARBS yet. Also, joypixels is not the full emojione, and didn't solve the issue. I didn't know about Nerdfont, but twemoji solved the issue for me.

**Comment by kronikpillow** [comment]
mate the issue is solved, I was the first one to open up this issue about ST crashing as soon as you touch the emoji or statusbar files, the issue is solved by installing ttf-joypixels ttf-symbola and nerd-fonts-complete ... nerd fonts complete are for the lf icons, and some icons in the status bars, and the rest is emoji coverage ... it's solved in #413

**Comment by Jemi** [comment]
Yes, this is not a voidrice issue, this is a larbs issue. That's why I opened a bug there. I did search and find related bugs in LARBS that were already closed.  But this is a new install and those bugs were old. I didn't find the open voidrice bugs although I did search. ttf-symbola won't do for LARBS because licensing and distribution issues will only lead to more problems later. twemoji solved the problem for me and is available in aur. I have no idea if nerdfonts solves the problem. ttf-symbola does not solve the problem because using a package that users aren't even supposed to be able to get does not solve any problem, it just creates problems later.
